python/composio/tools/toolset.py

```diff
@@ -67,22 +67,28 @@
 P = te.ParamSpec("P")
 
 _KeyType = t.Union[AppType, ActionType]
-_ProcessorType = t.Callable[[t.Dict], t.Dict]
+_CallableType = t.Callable[[t.Dict], t.Dict]
 
 MetadataType = t.Dict[_KeyType, t.Dict]
 ParamType = t.TypeVar("ParamType")
 
+# Enable deprecation warnings
+warnings.simplefilter("always", DeprecationWarning)
```

Comment:
Consider adding a type hint for the `warnings` module to improve code readability and maintainability.